### ISSUE TITLE
[bug]: Prompt ignored / random image on 3rd party safetensor model

### ISSUE THREAD

**Issue by furiousjay** [invoke-ai/InvokeAI#2738]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

macOS

### GPU

mps

### VRAM

8gb

### What happened?

Created Images on InvokeAI are completely random and seem to ignore the prompt completely when I use a 3rd party Safetensor Model. It just produces random images.

- If I use the integrated stable-diffusion model, the image is created with the prompt. 
- If I use the same Model on Automatic1111, it is working
- I use a MacBook Pro 2021 16"
- I added screenshots and the termin

### Screenshots

https://imgur.com/a/HAww6OI

### Additional context

Please enter 1, 2, 3, 4, 5, 6 or 7: [2] 2
Starting the InvokeAI browser-based UI...
* Initializing, be patient...
>> Initialization file /Users/joscha/invokeai/invokeai.init found. Loading...
>> Internet connectivity is True
>> InvokeAI, version 2.3.0
>> InvokeAI runtime directory is "/Users/joscha/invokeai"
>> GFPGAN Initialized
>> CodeFormer Initialized
>> ESRGAN Initialized
>> Using device_type mps
>> xformers not installed
>> Current VRAM usage:  0.00G
>> Loading diffusers model from runwayml/stable-diffusion-v1-5
  | Using more accurate float32 precision
  | Loading diffusers VAE from stabilityai/sd-vae-ft-mse
  | Using more accurate float32 precision
Fetching 15 files: 100%|█████████████████████| 15/15 [00:00<00:00, 77768.31it/s]
  | Default image dimensions = 512 x 512
>> Model loaded in 16.25s
>> Textual inversions available: 
>> Setting Sampler to k_lms (LMSDiscreteScheduler)

* --web was specified, starting web server...
* Initializing, be patient...
>> Initialization file /Users/joscha/invokeai/invokeai.init found. Loading...
>> Started Invoke AI Web Server!
>> Default host address now 127.0.0.1 (localhost). Use --host 0.0.0.0 to bind any address.
>> Point your browser at http://127.0.0.1:9090
>> System config requested
>> patchmatch.patch_match: INFO - Compiling and loading c extensions from "/Users/joscha/invokeai/.venv/lib/python3.10/site-packages/patchmatch".
>> patchmatch.patch_match: ERROR - patchmatch failed to load or compile (Command 'make clean && make' returned non-zero exit status 2.).
>> patchmatch.patch_match: INFO - Refer to https://invoke-ai.github.io/InvokeAI/installation/060_INSTALL_PATCHMATCH/ for installation instructions.
>> Patchmatch not loaded (nonfatal)
>> System config requested
>> System config requested
>> Model change requested: realisticVisionV13_v13VAEIncluded
>> Current VRAM usage:  0.00G
>> Offloading stable-diffusion-1.5 to CPU
>> Loading realisticVisionV13_v13VAEIncluded from /Users/joscha/stable-diffusion-webui/models/Stable-diffusion/realisticVisionV13_v13VAEIncluded.safetensors
>> Scanning Model: realisticVisionV13_v13VAEIncluded
>> Model scanned ok!
>> Loading realisticVisionV13_v13VAEIncluded from /Users/joscha/stable-diffusion-webui/models/Stable-diffusion/realisticVisionV13_v13VAEIncluded.safetensors
   | Forcing garbage collection prior to loading new model
   | LatentDiffusion: Running in eps-prediction mode
   | DiffusionWrapper has 859.52 M params.
   | Making attention of type 'vanilla' with 512 in_channels
   | Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
   | Making attention of type 'vanilla' with 512 in_channels
   | Using more accurate float32 precision
>> Model loaded in 31.26s
>> Model loaded in 31.48s
>> Textual inversions available: 
>> Setting Sampler to k_lms

>> Image Generation Parameters:

{'prompt': 'a green car', 'iterations': 1, 'steps': 50, 'cfg_scale': 7.5, 'threshold': 0, 'perlin': 0, 'height': 512, 'width': 512, 'sampler_name': 'k_euler', 'seed': 493816845, 'progress_images': False, 'progress_latents': True, 'save_intermediates': 5, 'generation_mode': 'txt2img', 'init_mask': '...', 'seamless': False, 'hires_fix': False, 'variation_amount': 0}

>> ESRGAN Parameters: False
>> Facetool Parameters: False
>> Setting Sampler to k_euler
/Users/joscha/invokeai/.venv/lib/python3.10/site-packages/ldm/modules/embedding_manager.py:146: UserWarning: The operator 'aten::nonzero' is not currently supported on the MPS backend and will fall back to run on the CPU. This may have performance implications. (Triggered internally at /Users/runner/work/pytorch/pytorch/pytorch/aten/src/ATen/mps/MPSFallback.mm:11.)
  placeholder_rows, placeholder_cols = torch.where(
Generating:   0%|                                         | 0/1 [00:00<?, ?it/s]>> Ksampler using model noise schedule (steps >= 30)
>> Sampling with k_euler starting at step 0 of 50 (50 new sampling steps)
100%|███████████████████████████████████████████| 50/50 [01:18<00:00,  1.56s/it]
100%|███████████████████████████████████████████| 50/50 [01:18<00:00,  1.27s/it]

>> Image generated: "/Users/joscha/invokeai/outputs#/000003.1e3d309d.493816845.png"

Generating: 100%|█████████████████████████████████| 1/1 [01:22<00:00, 82.63s/it]

>> Usage stats:
>>   1 image(s) generated in 83.03s

**Comment by lstein** [comment]
Hi there! I have been able to reproduce your issue with this particular safetensors model. As far as I'm aware this is not a general issue with other safetensors models but please post followups if you find models with similar problems.

==Here is what to do to work around the problem==
1. Open up the command line client by choosing item 1 from the invoke launcher
2. After the `invoke>` prompt appears, type `!optimize_model realisticVisionV13_v13VAEIncluded`
3. This will convert the model to `diffusers` format. When I tried the diffusers version, it worked fine.

**Comment by lstein** [comment]
There's no followup on this so I'm hoping this means the workaround was acceptable.
